src/App.js

```diff
@@ -2,70 +2,122 @@ import {nanoid} from 'nanoid';
 import {useState} from 'react';
 import styled from 'styled-components';
 
+import Footer from "./components/Footer";
+import Header from "./components/Header";
 import InputDataDialog from './components/InputDataDialog.js';
 import { initialInputData } from "./db";
+import { messages } from "./db";
 
 export default function App() {
   
   const [inputs, setInputs] = useState(initialInputData);
+  //const [inputValues, setInputValues] = useState(inputs[0].value);
+  const [showmessages, setShowMessages] = useState(messages[0].text);
```

Comment:
Using the name `showMessages` suggests that the state is an array with multiple messages (plural). However, you only use the text of the first message and therefore this is a string. The naming is a bit off then. Ask yourself, what you would like to archive here and what `showMessages` really is and change it accordingly.
```suggestion
  const [showMessages, setShowMessages] = useState(messages[0].text);
```